### PR TITLE
add Microsoft talk to resources and 'user stories' on the landing page

### PR DESCRIPTION
**Pull request by dholbach**
cc @staceypotter 

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
→ https://deploy-preview-468--fluxcd.netlify.app/
→ https://deploy-preview-468--fluxcd.netlify.app/resources

File: content/en/endorsements.yaml [hunk 1]
```diff
@@ -82,3 +82,17 @@ endorsements:
...
       <br/><br/>
       Michael Medellin & Gordon Tillman, <b>Department of Defense</b>
+  - image: https://i3.ytimg.com/vi/dtGgOVYJTv8/0.jpg
+    text: >
+      <div class="summary">
+        <div class="business">
+          <a href="https://www.youtube.com/watch?v=dtGgOVYJTv8">GitOps Days 2021 Keynote: GitOps in Microsoft Azure with Flux - Chris Sanders & Jonathan Innis</a>
+        </div>
+        <div class="details">
+          In this session we will show how CNCF Flux is enabled in Azure Arc enabled Kubernetes and Azure Kubernetes Services and also give a sneak peek at implementation of Flux v2.
```
Comment (staceypotter) [inline content/en/endorsements.yaml:92]: remove "will" - change to: "In this session we show how..."

**Review by staceypotter — COMMENTED** [review]
Just one small change, otherwise looks great! Thanks @dholbach ❤️‍🔥

**Review by hiddeco — APPROVED** [review]
Nice resources page :100: 

One note for the SEO of the page: it would be better to make the `h2` an `h1`, and the `h3` an `h2`.

**Pull request merged**